### DISCUSSION TITLE
Forwarder node with lora.

### DISCUSSION THREAD

**Discussion by NuwanUdara** [omnetpp/omnetpp#1097] [Q&A]
I am hoping to create a multi-hop Non-LoRaWAN network with simple LoRaNode devices. 
What I intend to do is, when an end device send a packet, the closest EndNodes and Relay devices should receive the packet if they are using the same Spreading factor and other channel states. If a relay device was able to listen that packet, then it should forward the packet to next nodes in its broadcast range.

Can someone share any work they have done with handling a Downlink in LoRaNodes, and LoRa packet forwarders. Since to my knowledge FloRa Does not support this kind of mesh network.

Thank you.

**Comment by aarizaq** [comment]
A multihop lora completely breaks the objective of lora. You will need four addresses in the packet, the source and destination address, the address of the intermediate packet, and the sender, it is similar to 802.11s. Also, you need to include a routing table and a routing protocol.
